client/client_unix.go

```diff
@@ -1,4 +1,4 @@
-// +build linux freebsd openbsd darwin
+// +build linux freebsd openbsd darwin solaris illumos
```

Comment:
Can we change this to `!windows`?

Reply:
Theoretically yes. Which is what _unix of the filename implies. However, I would look into all other files which have a _unix suffix and change them too. 

Considering that other unix system will want to compile consumers like traefik I would be willing to make the effort to change all _unix suffixed files to follow the go teams recommendations on this topic in a separate PR.